text/0630-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: https://github.com/emberjs/rfcs/pull/630
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
+
+## Summary
+
+> In dropping support for IE11 at some point in the future, we will need a new
+browser support policy. This is the proposed policy.
+
+## Motivation
+
+> It should be clear which browsers are supported by Ember.js
+
+## Detailed policy
+
+- Desktop
+
+    1. Google Chrome
+
+        - The current and previous stable releases are supported
+
+    2. Mozilla Firefox
+
+        - The current and previous rapid releases and any [ESR](https://support.mozilla.org/en-US/kb/choosing-firefox-update-channel) versions currently supported by Mozilla are supported
+
+    3. MS Edge
+
+        - The current and previous stable releases are supported.
+
+    4. Mac Safari
+
+        - The versions coming with the current and previous releases of MacOS are supported, as well as versions of Mac Safari available with updates to the current release.
+
+- Mobile
+
+    1. Google Chrome
+
+        - The versions coming with all versions of Android released in the past 36 months are supported, as well as the current and most stable releases available via update.
+
+    2. iOS Safari
+
+        - The versions coming with all versions of iOS released in the past 36 months are supported.
+
+    3. Android
+
+        - The versions released with all versions of Android released in the past 36 months are supported.
+
+All browsers not listed here may work but are not explicitly supported.
+
+## How we teach this
+
+> A blog post explaining the policy with a link to this RFC will be posted.
+
+## Drawbacks
+
+- See discussions on this RFC
+
+## Alternatives
+
+> See discussions on this RFC
```

Comment:
```suggestion
- Things stay as they are now
- We include assistive technology (largely unchanged in the past 25+ years) in our support policy

The discussion/comments may identify additional alternatives that have not been identified yet; they will be added to this RFC if that is the case. 
```

Reply:
I think assistive technology support should be covered in a separate RFC.

Reply:
I disagree, I think saying "accessibility is different" is the root cause of accessibility issues on the web for the past 30+ years the web has existed. We can do better.

Reply:
I agree that "accessibility is not different". I disagree that I should be the one to propose the standards, due to my ignorance. :) If you'd like to write something up, I will try to add it, but I recommend that you write the RFC for an accessibility policy yourself, and I can link to it.

Reply:
I do not think adding "we include assistive technology" should be an *alternative*. It should be first class.